ansible/roles/loadbalancer/tasks/main.yml

```diff
@@ -0,0 +1,31 @@
+---
+# This role will install Loadbalancer in group 'loadbalancers' in the environment inventory
+
+- name: pull the lastest image of controller
+  shell: "docker pull {{ docker_registry }}whisk/balancer:{{ docker_image_tag }}"
+  when: docker_registry != ""
+
+- name: (re)start loadbalancer
+  docker:
+    name: controller
+    image: "{{ docker_registry }}whisk/controller:{{ docker_image_tag }}"
+    state: reloaded
+    restart_policy: "{{ docker.restart.policy }}"
+    log_driver: syslog
+    env:
+      "COMPONENT_NAME": "loadbalancer"
+      "CONSULSERVER_HOST": "{{ groups['consul_servers']|first }}"
```

Comment:
same as controller. putting localhost should work. (i.e. local docker host, not "localhost")